### PR TITLE
Closes #424 - Modify fluo script to load a user defined fluo-env.sh conf...

### LINKED ISSUE #424: Modify fluo script to load a user defined fluo-env.sh config file
Fluo used to have a configuration file called fluo-env.sh but it was removed because there was no use for it.  However, after working on the fluo-deploy script, fluo-env.sh is needed again.  

The new fluo-env.sh would be loaded by the fluo script.  This would prevent users from setting $HADOOP_PREFIX in their .bashrc.

### PR DESCRIPTION
**Pull request by mikewalch**
...ig file

### CODE REVIEW

File: modules/distribution/src/main/scripts/fluo [hunk 1]
```diff
@@ -25,6 +25,7 @@ bin="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 script=$( basename "$SOURCE" )
 # Stop: Resolve Script Directory
 
+. "$bin"/../conf/fluo-env.sh
```
Comment (keith-turner) [inline modules/distribution/src/main/scripts/fluo]: could config.sh source fluo-env.sh?

File: modules/distribution/src/main/scripts/impl/init.sh [hunk 2]
```diff
@@ -32,6 +32,11 @@ if [[ -z $HADOOP_PREFIX ]]; then
   exit 1
 fi
 
+if [ ! -d "$HADOOP_PREFIX" ]; then
```
Comment (keith-turner) [inline modules/distribution/src/main/scripts/impl/init.sh]: could this check be done in config.sh, if config.sh sourced fluo-env.sh?

Comment (mikewalch) [inline modules/distribution/src/main/scripts/impl/init.sh]: I initially had it there but `fluo mini` should not require that `$HADOOP_PREFIX` is set and this only needed for the `init` & `yarn` commands

File: modules/distribution/src/main/scripts/fluo (hunk 1 above)
Comment (mikewalch) [inline modules/distribution/src/main/scripts/fluo]: Yes it could.  I will change.

File: modules/distribution/src/main/scripts/impl/init.sh (hunk 2 above)
Comment (mikewalch) [inline modules/distribution/src/main/scripts/impl/init.sh]: fixed in b72d3fc34b34

Comment (keith-turner) [inline modules/distribution/src/main/scripts/impl/init.sh]: > I initially had it there but fluo mini should not require that $HADOOP_PREFIX

nice.  Thats very user friendly

**Comment by keith-turner** [comment]
+1

**Pull request merged**